mobile_files/package.json.orig

```diff
@@ -65,7 +65,7 @@
     "react-native-webview": "^5.2.1",
     "react-native-webview-bridge": "git+https://github.com/status-im/react-native-webview-bridge.git#fix/classnames-colision",
     "react-navigation": "^2.12.1",
-    "realm": "2.28.0",
+    "realm": "^2.28.1",
```

Comment:
I think we're tending towards using specific version numbers (and upgrade explicitly as necessary). At least that's what I've been seeing @yenda doing.

Reply:
Ah, you're right, I didn't notice the `^`. @yenda can you please drop it?

Reply:
yes I will I just want a quick build to check if it fixes the buck, it's stupid yarn that did that, is there a way to make it upgrade a lib without acting out like this?

Reply:
`yarn upgrade --exact`?

Reply:
It's still using the `^`.